debian/control

```diff
@@ -25,7 +25,7 @@ Depends:
     python-apt,
     schedtool,
     python-pip,
-    kano-i18n (>= 3.5-1),
+    kano-i18n (>= 3.15-1),
```

Comment:
*3.15.0

Reply:
Yes - or at least, corresponding to kano-i18n.